extra_data/cli/make_virtual_cxi.py

```diff
@@ -67,8 +80,13 @@ def main(argv=None):
     )
     ap.add_argument(
         '--exc-suspect-trains', action='store_true',
-        help="Exclude suspect trains. This tries to avoid some issues with incorrect train IDs in the data, "
-             "but may mean less data is available."
+        help='Exclude suspect trains. This tries to avoid some issues with'
+             ' incorrect train IDs in the data, but may mean less data is'
+             ' available.'
+    )
+    ap.add_argument(
+        '-seq', '--per-sequence', action='store_true',
+        help='Create separate CXI files for each data sequence.'
     )
```

Comment:
Are we confident this is actually useful? My memory of the users doing something similar before was that they were making each CXI file as the corresponding sequence was ready, so they'd want the ability to pass in a sequence number, rather than make CXI files for all sequence numbers in the directory.

If we're not pretty sure it's what users want, let's leave it out for now. Once the machinery is ready, it's easy to write a custom script to do something like this. Extra options have a cost both for users (more to read and figure out) and us (more to test and support).

Reply:
Sure, thank you, removed in https://github.com/European-XFEL/EXtra-data/commit/5b0b07650f36da21278bb8f04a32cbc701e2cc24.